woocommerce/google-listings-and-ads
# Multi-country targeting campaign - Move the free ads credit box to step 1 of ads account setup

### Changes proposed in this Pull Request:

This PR implemented the "Move the free ads credit box to step 1 of ads account setup" part of #1248:

- Move the free ads credit box to step 1 of ads account setup.
- Move the *free-ad-credit* folder and adjust its layout and style to fit with the visual design.

:warning: Some notes:
- It won't be shown on the single-page campaign creation/edit page. (Confirmed with Jill)
- The copies have been changed after https://github.com/woocommerce/google-listings-and-ads/pull/1217 and the updated copies should be the latest. So the copies in the box will be different between implementation and visual design.

### Screenshots:

https://user-images.githubusercontent.com/17420811/154460450-ce068338-8420-4f1a-8d99-8862bb53ecc8.mp4

### Detailed test instructions:

💡  If test with an unconnected account, it will encounter a bug that could be addressed by #1252. It will need to cherry-pick fix from #1252 on your local if want to test from an unconnected ads account. 🙏 

Or, since this PR hasn't changed any logic, it could be tested by simulating a connected Google ads account that fulfills the eligibility of free ads credit. To do so, add the following code snippet at [line 26 of useFreeAdCredit hook](https://github.com/woocommerce/google-listings-and-ads/blob/e396335223d453a1d0d4885ebb6c6a471fc5abee/js/src/hooks/useFreeAdCredit.js#L26).
```js
if ( googleAdsAccount?.status === 'connected' ) {
	return true;
}
```

1. Go to ads account setup: `/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fsetup-ads`
2. The free ads credit box should appear after Google ads account data is fetched and its status is connected.
3. Continue to step 2. There should be no free ads credit box anymore.
4. Go to the campaign creation/edit page. There should be no free ads credit box anymore.

### Changelog entry
